src/backend/DocDownloaderRedisCache/src/main/java/ca/bc/gov/ag/courts/util/JobValidator.java

```diff
@@ -0,0 +1,33 @@
+package ca.bc.gov.ag.courts.util;
+
+import ca.bc.gov.ag.courts.model.Job;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class JobValidator {
+    private static final String DATE_FORMAT_PATTERN = ".+[+-]\\d{2}:\\d{2}$";
+    private static final String DATE_FORMAT_ERROR = "Date is not in ISO8601 format with offset";
```

Comment:
Would you mind adding an example of a good time/date format with offset in the error messages. Thanks Peiman

Reply:
added more tests see in test class: JobValidatorTest

Reply:
Because time / date format can be a pain - What I am asking for here is to include the format in DATE FORMAT error strings. 
For example: "Date is not in ISO8601 format with offset. Ex: 2024-01-19T19:00-07:00"